db/migrate/20230621072445_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
:red_circle: As [per this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md) Your blog-app should implement the `:confirmable` module. Please enable and implement this module as it is one of  the project requirements

![requi](https://github.com/maov19/blog-app/assets/26736582/2dd07f53-a5cc-41c5-8ade-972e73063d97)

Reply:
Fixed! Added the confirmable module, as required, both in migration file and models.